traefik.go

```diff
@@ -79,17 +86,25 @@ func main() {
 
 	// listen new configurations from providers
 	go func() {
+
 		for {
-			configuration := <-configurationChan
-			log.Infof("Configuration receveived %+v", configuration)
-			if configuration == nil {
+			configMsg := <-configurationChan
+			log.Infof("Configuration receveived from provider %v: %+v", configMsg.providerName, configMsg.configuration)
+			if configMsg.configuration == nil {
 				log.Info("Skipping empty configuration")
-			} else if reflect.DeepEqual(currentConfiguration, configuration) {
```

Comment:
I'm not sure about that... If we have an first conf update from file provider and then a second from docker provider, this test will compare the previous file config with the new docker one.